tools/downloader/caffe2_to_onnx.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+from pathlib import Path
+import sys
+import os
+import re
+
+import onnx
+from caffe2.python.onnx.frontend import Caffe2Frontend
+from caffe2.proto import caffe2_pb2
+
+
+def positive_int_arg(values):
+    """Check positive integer type for input argument"""
+    result = []
+    shapes = re.findall(r'[(\[]([0-9, -]+)[)\]]', values)
+    for shape in shapes:
+        single_shape = []
+        for value in shape.split(','):
+            try:
+                ivalue = int(value)
+                if ivalue < 0:
+                    raise argparse.ArgumentTypeError('Argument must be a positive integer')
+                single_shape.append(ivalue)
+            except Exception as exc:
+                print(exc)
+                sys.exit('Invalid value for input argument: {!r}, a positive integer is expected'.format(value))
+        result.append(single_shape)
+    return result
+
+def parse_args():
+    """Parse input arguments"""
+
+    parser = argparse.ArgumentParser(description='Conversion of pretrained models from Caffe2 to ONNX')
+
+    parser.add_argument('--model-name', type=str, required=True,
+                        help='Model to convert. May be class name or name of constructor function')
+    parser.add_argument('--output-file', type=Path, required=True,
+                        help='Path to the output ONNX model')
+    parser.add_argument('--model-path', type=str, required=True,
+                        help='Path to predict_net .pb file')
+    parser.add_argument('--weights', type=str, required=True,
+                        help='Path to init_net .pb file')
```

Comment:
The names of these parameters seem to be unnecessarily obscure. Why not call them `--predict-net` & `--init-net`, or something along those lines?

Reply:
It is done for consistent with `pytorch_to_onnx.py` parameter name.

Reply:
But they don't have the same semantics as those parameters. Why should they be consistent with them?

Reply:
Actually the semantics is quite the same, `init-net` stores all the weights, offsets, etc. used by the network, while the `predict-net` stores operations that that execute the net.